src/components/form/HdTextarea.vue

```diff
@@ -24,6 +24,7 @@
       :style="{ height }"
       :disabled="disabled"
       class="textarea"
+      data-gramm_editor="false"
```

Comment:
I would recommend you to add a comment here (or just creating a data where comment would be allowed) explaining this property. If we refactor and loose this commit reference it will be hard to back track.

Reply:
Good point, I'll add a comment.